\chapter{Implications of codon–anticodon interaction on the regulation of translation}

In the previous chapter I have shown that the codon usage and its interaction
with \trna anticodons remains remarkably stable across the variability of the
transcriptome during mammalian development.

Shortly after the publication of our research on \trna gene regulation during
mouse development, \citet{Gingold:2014} published \texttitle{A Dual Program for
Translation Regulation in Cellular Proliferation and Differentiation}. They
report that different programmes of cellular function preferentially use
different sets of codons, and that the pool of active \trna[s] adapts
dynamically to decode this set of codons with high efficiency.

In order to bring in line these two findings — the stability of the anticodon
pool on the one hand, and the malleability of the anticodon pool to match demand
of highly expressed on the other hand — we turned our attention to differences
between healthy tissue and tumour cell lines in \mmu and \hsa.

\todo[inline]{Summarise \citet{Gingold:2014} here}

\citet{Gingold:2014} investigated the abundance of \trna anticodons and the
codon usage of different groups of protein-coding genes in human-derived cell
lines in different cellular conditions --- N\todo{find number} tumour tissues,
induced differentiation, release from serum starvation and senescence.

Using microarray expression data, they show that there are specific anticodon
isoacceptors whose abundance changes reproducibly in specific tumours
(lymphomas), mirroring results reported previously\todo[ref]{Canella? White?}.
They then looked at genes of \go terms which they associated with healthy, adult
tissue (“pattern specification process”) and tumour tissue (“M phase of mitotic
cell cycle”). They show that the codon usage bias in these two \go terms
(averaged over all containing genes) clearly differs\todo{fig 2A?}.

They expand their analysis by calculating the mean codon usage bias across many
categories of \go terms and performing \pca on them. This reveals that the
largest contributor to the variation stems from the split of the \go terms into
two distinct sets, one encompassing multi-cellular \go terms and the other cell
autonomous \go terms. They argue that these two sets of \go terms correspond to
\go terms functionally responsible for maintaining cell homoeostasis on the one
hand, and rapid cellular division, such as as found in tumours, on the other
hand. In other words, different sets of codons are preferentially used in
rapidly dividing cells than in healthy cells.

\textfig{go-cub-pca}{body}{0.8\textwidth}
    {\pca of mean \go term codon usage bias}
    {\todo{Add description}. Created using methods of \citet{Gingold:2014}}

\todo[inline]{Include table of two sets of GO terms referred to above}

Finally, \citet{Gingold:2014} use \rnaseq data of the previously mentioned cell
types to argue that gene expression changes do indeed correlate with the first
principal component of the \pca, with tumour samples showing more increased
expression towards the more we go towards the cell autonomous end of the axis,
and more decreased expression towards the multi-cellular end, and that this
trend is reversed for samples taken after induced differentiation using retinoic
acid. A similar analysis is then done for the expected translational efficiency,
by using \trna anticodon abundance to calculate the fold change of the \tai,
matched against the codon usage bias in the different \go terms. However, the
authors neglected to actually calculate correlations between the first principal
component and either the codon usage bias or the \tai fold change. Instead, they
merely visualised the presumed correspondence using an inadequate and
(particularly in this case) misleading colour map \citep{Borland:2007}.
Furthermore, the absolute range of changes are minute (range of \tai fold change
\numrange{0.86}{0.905} in one case), and lack of statistical analysis makes it
impossible to say whether these changes are in fact significant, assuming they
correlate at all.

Since we were already in possession of relevant \trna data we decided to use our
own data to recapitulate these findings, rather than merely using the data from
\citet{Gingold:2014}. In addition, since we were not convinced by the
statistical analysis presented in their paper, we decided to address this as
well.

A first observation was that while our own research so far had looked at the
whole transcriptome, \citet{Gingold:2014} had looked at specific subsets. While
both approaches are valid in their own right, some caution is necessary when
comparing the results: the smaller the set of genes one looks at, the larger the
effects of random sampling of the genes become. When analysing a particular
feature, such as the codon usage bias, random sampling will thus contribute a
larger part to the variation between two small sets than between two large sets.
If one wants to assert that deviations are nonrandom, one thus has to account
for this effect.

\section{Challenges in the interpretation of codon–anticodon interaction}

The \pca of the codon usage across \go terms struck me as questionable for two
reasons:

\begin{enumerate}
    \item The selection of the \go terms for the two categories
        \emph{multi-cellular} and \emph{cell autonomous}, while not entirely
        arbitrary, is very open to non-objective criteria. For instance, genes
        with functionally opposite influences on a given biological process may
        occur in the same \go term. Furthermore, it is not clear how the two
        categories relate to the phenotypes \emph{proliferation} and
        \emph{differentiation}.
    \item The size of the \go-associated gene sets (starting at \num{40} genes)
        is small enough so that random variation in the codon usage on the gene
        level will have a noticeable influence on the distribution.
\end{enumerate}

Summarise our objections:

\todo[inline]{\begin{enumerate}
    \item We find a stable codon and \trna pool, no adaptation to specific cellular
conditions

    \item Their analysis does not exclude alternative explanations (which I partially
(?) list here)

        \begin{enumerate}
            \item size of \mrna subsets is a strong covariant, with small subsets
                showing larger variation on codon usage bias, purely due to
                sampling variation
            \item correlation is not just with codon usage but also amino acid
                usage. How is that possible if the codons control translation?
            \item evolutionarily, codon bias does not seem conserved, which
                makes it hard to think of it as a conserved driver of regulation
            \item perfect correlation of codon usage and GC bias, may hint at
                the existence of a more fundamental relationship on the
                transcriptional level
            \item failing adaptation of \emph{anti}codon pool poses big problem
                for theory that subsets of \mrna may be adapted to
        \end{enumerate}

    \item Using histone marks as proxy for tRNA expression is extremely noisy
\end{enumerate}}

\section{Specific \abbr{trna} transcriptomes are not specifically adapted to the
codon usage of highly expressed \abbr{mrna} transcripts}

\section{There is no evolutionary conservation of codon selection in mammals}
