\chapter{Implications of codon–anticodon interaction on the regulation of translation}

\section{Codon usage differs in functionally meaningful subsets of the
transcriptome in mammals}

\todo{Summarise \citet{Gingold:2014} here}

\section{Challenges in the interpretation of codon–anticodon interaction}

Summarise our objections:

\begin{enumerate}
    \item We find a stable codon and \trna pool, no adaptation to specific cellular
conditions

    \item Their analysis does not exclude alternative explanations (which I partially
(?) list here)

        \begin{enumerate}
            \item size of \mrna subsets is a strong covariant, with small subsets
                showing larger variation on codon usage bias, purely due to
                sampling variation
            \item correlation is not just with codon usage but also amino acid
                usage. How is that possible if the codons control translation?
            \item evolutionarily, codon bias does not seem conserved, which
                makes it hard to think of it as a conserved driver of regulation
            \item perfect correlation of codon usage and GC bias, may hint at
                the existence of a more fundamental relationship on the
                transcriptional level
            \item failing adaptation of \emph{anti}codon pool poses big problem
                for theory that subsets of \mrna may be adapted to
        \end{enumerate}
\end{enumerate}

\section{Specific \abbr{trna} transcriptomes are not specifically adapted to the
codon usage of highly expressed \abbr{mrna} transcripts}

\section{There is no evolutionary conservation of codon selection in mammals}
