\chapter{Implications of codon–anticodon interaction on the regulation of translation}

In the previous chapter I have shown that the codon usage and its interaction
with \trna anticodons remains remarkably stable across the variability of the
transcriptome during mammalian development.

Shortly after the publication of our research on \trna gene regulation during
mouse development, \citet{Gingold:2014} published a paper titled \texttitle{A
Dual Program for Translation Regulation in Cellular Proliferation and
Differentiation}. They report that different programmes of cellular function
preferentially use different sets of codons, and that the pool of active
\trna[s] adapts dynamically to decode this set of codons with high efficiency.
\citet{Gingold:2014} demonstrated this in several different tumour cell lines.

In order to bring in line these two findings — the stability of the anticodon
pool on the one hand, and the malleability of the anticodon pool to match demand
of highly expressed on the other hand — we turned our attention to differences
between healthy tissue and tumour cell lines in \mmu and \hsa.

\todo{Summarise \citet{Gingold:2014} here}

\section{Challenges in the interpretation of codon–anticodon interaction}

The \pca of the codon usage across \go terms struck me as questionable for two
reasons:

\begin{enumerate}
    \item The selection of the \go terms for the two categories
        \emph{multi-cellular} and \emph{cell autonomous}, while not entirely
        arbitrary, is very open to non-objective criteria. For instance, genes
        with functionally opposite influences on a given biological process may
        occur in the same \go term. Furthermore, it is not clear how the two
        categories relate to the phenotypes \emph{proliferation} and
        \emph{differentiation}.
    \item The size of the \go-associated gene sets (starting at \num{40} genes)
        is small enough so that random variation in the codon usage on the gene
        level will have a noticeable influence on the distribution.
\end{enumerate}

Summarise our objections:

\begin{enumerate}
    \item We find a stable codon and \trna pool, no adaptation to specific cellular
conditions

    \item Their analysis does not exclude alternative explanations (which I partially
(?) list here)

        \begin{enumerate}
            \item size of \mrna subsets is a strong covariant, with small subsets
                showing larger variation on codon usage bias, purely due to
                sampling variation
            \item correlation is not just with codon usage but also amino acid
                usage. How is that possible if the codons control translation?
            \item evolutionarily, codon bias does not seem conserved, which
                makes it hard to think of it as a conserved driver of regulation
            \item perfect correlation of codon usage and GC bias, may hint at
                the existence of a more fundamental relationship on the
                transcriptional level
            \item failing adaptation of \emph{anti}codon pool poses big problem
                for theory that subsets of \mrna may be adapted to
        \end{enumerate}
\end{enumerate}

\section{Specific \abbr{trna} transcriptomes are not specifically adapted to the
codon usage of highly expressed \abbr{mrna} transcripts}

\section{There is no evolutionary conservation of codon selection in mammals}
