\chapter{Conclusion}

\dictum{… except in the light of genetics.\par\dictumrule}

\noindent
Evolution by natural selection is the implicit assumption underlying all of
modern biology. \citet{Dobzhansky:1973} famously argues against ill-informed
criticism of evolution, stating

\begin{quote}
    Nothing in biology makes sense except in the light of evolution.
\end{quote}

This statement is as relevant today as it was then --- both as an admonishment
of ignorance of basic biological facts, and as serving as a summary of our
understanding. In fact, modern evolutionary synthesis, which is the prevailing
explanatory model employed today, and which is itself an evolution of the
Darwinian theory of descent with modification by means of natural selection, is
unchallenged in this status, and is corroborated by every new piece of evidence.

In \texttitle{The making of the fittest}, Sean Carroll argues that the best
evidence for evolution we have is the genetic record which we can read in the
\dna of the living species \citep{Carroll:2006}. Carroll was mainly talking
about the similarity between homologous genes in different species. But one of
the most striking examples of strong conservation, and which defies explanation
when not acknowledging common descent, is the near-universal genetic code. In
billions of years of evolution, the instruction set used to encode the building
blocks of proteins has remained virtually unchanged.

\todo[inline]{… waffle, waffle, genetic code is well-preserved, tRNA--mRNA
interaction is physical manifestation of said code. Great evidence for
evolution, and its stability underlies the viability of cells (ergo disruptions
lead to defects which are clinically relevant, cf. cancer}

\section{Future directions}

The question of what causes codon usage bias variability across functional
subsets of the transcriptome remains wide open. \gc bias, in particular, is
worth exploring further. On the one hand, I observed a robust correlation
between \gc bias and codon usage, and we know that prokaryotic codon usage can
sometimes be predicted from intergenic \gc bias \citep{Chen:2004}. On the other
hand, \citet{Duret:2002} show that, at least in \species{dmel} and
\species{cel}, \gc bias is uncorrelated with codon usage bias.

To explore this further, two avenues seem obvious:

\begin{enumerate}
    \item \citet{Chen:2004} show that intergenic \gc predicts codon usage. Since
        intergenic regions are non-coding, this suggests that differential codon
        usage between genes has, at best, a minor functional relevance. So far,
        I have only looked at \gc bias in coding sequences. To make similar
        conclusions, I will have to instead compare codon usage to the \gc bias
        in the flanking regions of protein-coding genes.
    \item \gc bias can vary between synonymous and non-synonymous codons. If \gc
        bias is indeed causal for codon usage, we would expect that \gc bias
        correlates highly between the first two nucleotide positions of the
        codon and its wobble position. However, if the wobble position’s \gc
        bias is uncorrelated to the \gc bias of the other codon positions in a
        gene set, this would require a different explanation.
\end{enumerate}

Another feature known to constrain codon deployment is the presence of other
sequence features in the coding region of genes. This includes binding sites for
enhancers \citep{Blencowe:2000}.\todo{what to do about it?}

\todo[inline]{Conservation of codon usage and GC bias}

\todo[inline]{Use tAI for codon--anticodon adaptation analysis}
