\chapter{\abbr{rna} sequencing quantifies protein-coding gene expression}

\section{The transcriptome reflects the state of the cell}

Cells are complex, self-contained units of life, with a cell membrane made up of
a lipid bilayer, which defines a boundary between the outside world and
everything that belongs to the cell. Inside the cell, a highly complex machinery
made up of many parts keeps the cell alive.

As we have seen in the introduction, the central part of this machinery is
abstracted by the Central Dogma of molecular biology, with the \dna at one end
encoding the hereditary identity of the cell, and the proteins at the other end
as the effectors of this information.

It is therefore proteins which determine how a cell behaves, and the change of
proteins ultimately determines changes in cellular function. The entirety of
protein abundance in the cell at a given instance is called the
\define{proteome}. Unfortunately, quantifying the proteome in an unbiased
fashion is hard and expensive\todo{reference}. Instead, modern biology uses the
abundance of \mrna molecules, which code for individual proteins (the
\define{transcriptome}), as an accurate proxy of the proteome. The
appropriateness of this approach has been verified in numerous
studies\todo{citations}.

%\section{Regulation of protein-coding genes}

\section{Quantifying expression of \abbr{mrna} genes}

\subsection{Microarrays}

Historically, the abundance of \mrna has been determined using specific probes,
which hybridise to complementary target \mrna sequences. In order to determine
the abundance of many different \mrna[s] simultaneously, large arrays of
different probes can be generated and queried in parallel. Due to their
miniaturisation, these arrays are known as \define{microarrays}.

While microarrays are still playing an important role in transcriptome analysis,
they have recently been superseded by another technology due to two
disadvantages:
\begin{enumerate*}
    \item Each probe is sequence specific, and only recognises its target \mrna.
        As a consequence, quantification is inherently biased and requires that
        each target is known beforehand. Microarrays thus cannot discover new
        target transcripts, and the design of a new microarray design is
        technically challenging and expensive.\todo{reference}
    \item The signal from microarrays, which is a measure of each \mrna[â€™s]
        abundance in the sample, is sample-specific. This means that while we
        can compare the relative expression strength of two \mrna genes on the
        same microarray, comparing expression of genes between different
        microarrays is impossible.\todo{reference}
\end{enumerate*}

\subsection{\abbr{rnaseq}}

\section{Biological background of \abbr{rnaseq}}

\section{Computational methods to analyse \abbr{rnaseq} data}

\subsection{Expression normalisation}

\subsection{Differential expression}

\section{Using \abbr{rnaseq} to assay gene expression during mouse development}

\todo{This describes the specific method used in the \abbr{trna} paper}
In order to investigate protein-coding gene expression, we quantified the \mrna
abundance from \rrna-depleted \rnaseq data (strand-specific \SI{75}{bp}
paired-end reads from \name{Illumina} \name{HiSeq~2000}). Reads were mapped to
the \mmu reference genome (\abbr{ncbim37}) using \name{iRAP}
\citep{Fonseca:2014} and \name{TopHat2} \citep{Kim:2013}. Read counts were
quantified using \name{HTSeq} \citep{Anders:2014}, and assigned to
protein-coding genes from the \name{Ensembl} release \num{67}
\citep{Flicek:2014}.\todo{add \chipseq and \rnaseq pipeline flowcharts}

We excluded mitochondrial chromosomes from the analysis, because mitochondrial
genes use a slightly distinct genetic code\todo{ref}. Furthermore, we excluded
sex chromosomes.
