\chapter{Analysis challenges concerning \abbr{trna} expression data}
\label{sec:trna-analysis}

\section{Quantifying expression of \mrna genes}

In order to investigate protein-coding gene expression, we quantified the \mrna
abundance from \rrna-depleted \rnaseq data (strand-specific \SI{75}{bp}
paired-end reads from \name{Illumina} \name{HiSeq~2000}). Reads were mapped to
the \mmu reference genome (\abbr{ncbim37}) using \name{iRAP}
\citep{Fonseca:2014} and \name{TopHat2} \citep{Kim:2013}. Read counts were
quantified using \name{HTSeq} \citep{Anders:2014}, and assigned to
protein-coding genes from the \name{Ensembl} release \num{67}
\citep{Flicek:2014}.
\todo{add \chipseq and \rnaseq pipeline flowcharts}

We excluded mitochondrial chromosomes from the analysis, because mitochondrial
genes use a slightly distinct genetic code\todo{ref}. Furthermore, we excluded
sex chromosomes.

%\section{Normalisation of \trna gene expression}
%
%Raw \trna gene expression counts show strong differences between sequencing
%libraries. For \mrna count data, it is customary to use \name{DESeq}â€™s
%\dfn{library size normalisation} \citep{Anders:2010}. However, as shown in
%\cref{fig:trna-libraries-example}, library size normalised data still follows
%distinct distributions, which we deem biologically implausible, and probably due
%to technical bias.
%
%As a consequence, we opted to \dfn{quantile normalise} the count data\todo{find
%reference for quantile normalisation}.
%
%\textfloat{trna-libraries-example}{body}
%    {\centering\Huge Placeholder}{Placeholder}{Placeholder}

\section{Genomic clusters}

We defined \num{69} clusters of all genomically annotated \trna genes that lie
within \SI{7.5}{kb} of each other. We counted how many active \trna genes of an
isoacceptor family colocalised in a genomic cluster with \trna genes of the same
isoacceptor family. We calculated the fraction of \trna for each isoacceptor
family belonging to a genomic cluster. In order to test whether genes in
isoacceptor families tend to genomically colocalise more than expected by
chance, we randomly assigned \trna genes to isoacceptor families (preserving the
actual isoacceptor family gene numbers) \num{1000} times. We then tested whether
the mean percentage of clustering \trna genes per isoacceptor family differed
from the mean percentage expected by chance, by using a binomial test. Finally,
we tested whether there wa a difference in these percentages between isoacceptor
families that show evidence for compensation, and isoacceptor families that show
no such evidence by applying a \(\chi^2\)-test.

\section{An overview over failed analysis approaches}
