\chapter{Developmental stability of the \abbr{mrna}--\abbr{trna} interface}

In order to study how changes in \mrna gene expression relate to changes in
\trna gene expression, we collected tissue samples from six time points in mouse
(\mmu) development: two before birth (E15.5 and E18.5, which stands for
\num{15.5} and \num{18.5} days after gestation of the oocyte, respectively); two
shortly after birth, happening around E20 (P0.5 and P4 --- \num{0.5} and \num{4}
days after birth, respectively) and two after weaning the juvenile mice (P22 and
P29).

For each of these time points, tissue was collected from whole liver and whole
brain (homogenised) and prepared for \rnaseq and \pol3 \chipseq in order to
assay \mrna and \trna gene expression. The tissues were chosen for their
interesting shifts in physiology during development: the liver is a homogeneous
organ predominantly made up of a single cell type --- around \num{70} per cent
hepatocytes\todo{ref} --- and liver function changes fundamentally at birth;
prenatal liver serves mainly as a haematopoietic organ, whereas liver of
post-natal mice is primarily a metabolic organ.\todo{ref} Brain, by contrast, is
a highly heterogeneous organ made up of many different cell types, with dynamic
changes all through development.\todo{ref}

Each experiment was performed in two biological replicates, which were highly
correlated. \Cref{fig:trna-project-outline} summarises the experimental
procedure.\footnote{The wet-lab work of this project was performed by Bianca
Schmitt.} \todo{Remove background gradients from graphics}

\textfig{trna-project-outline}{body}{0.8\textwidth}
    {Sample analysis outline}
    {Samples were collected in eight distinct time points …}

In addition to the six time points described above, tissue was also collected at
two earlier stages, E9.5 and E12.5. Unfortunately, the embryo at such early
stages of development are too small to permit collecting enough tissue-specific
material. For that reason, we used the whole embryo at E9.5 and separated the
E12.5 embryo into torso and upper body. The subsequent analysis was performed on
the six later stages in liver and brain, unless noted otherwise.

The results presented in this chapter are published as \citet{Schmitt:2014}.

%\section{Mouse tissue development as a model system to study mRNA and tRNA gene
%regulation}

\section{Protein-coding gene expression changes dynamically during mouse
development}

Changes in the expression of protein-coding genes, leading to changes in
abundance of proteins, are known to drive cellular behaviour\todo{ref}. Our data
confirms that tissue development in mice is accompanied by large-scale changes
to the \mrna transcriptome (\cref{fig:mrna-expression-change}).

This is nicely illustrated by looking at individual gene expression patterns.
For example, we can confirm the functional relevance of apolipoprotein B
(\protein{APOB}) as the primary carrier of lipoproteins, which becomes
increasingly relevant as the organ shifts to metabolism \citep{Knott:1986}.
Similarly, \mrna gene expression changes highlight the role of α-fetoproteine
(\protein{AFP}) as the fetal version of serum albumin (\cref{fig:apob-afp})
\citep{Chen:1997}.\todo{Explain brain examples as well?}

A more systematic analysis shows that changes in gene expression happen
incrementally across the profiled stages of development: The \pca of the
pairwise correlation between all stages (and their gene expression values)
reveals that the first principal component is correlated with tissue identity,
and the second principal component is correlated with developmental time.
(\cref{fig:mrna-pca}). The number of differentially expressed genes between all
pairwise developmental stages shows a similar pattern, with more distinct
developmental stages having higher numbers of differentially expressed genes
(\cref{fig:mrna-de-matrix}).

These patterns are noteworthy because they recapitulate tissue identity and
linear progression through the stages of tissue development. But they are not
particularly surprising: after all, cell function is dictated by specific
protein abundance and thus protein-coding gene transcription --- the patterns of
gene expression similarity shown in the \pca therefore recapitulate the expected
changes in cell function between tissues, and over the course of development.

\textfloat{mrna-expression-change}{spill}{%
    \centering
    \begingroup
        \includegraphics[width=\textwidth]{liver-mrna-expression-change}
        \subcaption{\label{fig:apob-afp}Gene expression changes of \gene{Apob} and \gene{Afp}.}
    \endgroup
    \par
    \begingroup
        \includegraphics[width=\textwidth]{brain-mrna-expression-change}
        \subcaption{Gene expression changes of \gene{Foxp2} and \gene{Calm1}.}
    \endgroup}
    {Example of gene expression changes in development.}
    {The four genes are representative for tissue- and stage-specific genes,
    whose expression changes drive cell function. These changes can go up or
    down over the course of development, correspoding to either an up- or
    downregulation.}

\textfigtwo{mrna-pca}
    {\pca of \mrna gene expression per developmental stage.}
    {Rotations \num{1} and \num{2} of the correlation matrix of
    protein-coding gene expression in each developmental stage. The percentage
    on the axes shows the amount of variance explained by each rotation.}
    {mrna-de-matrix}
    {Number of differentially expressed \mrna genes between stages:}
    {Each off-diagonal square shows the number of differentially
    expressed genes (at a significance threshold of \(p<0.01\)) between
    the two indicated developmental stages.}

\section{Dynamic changes of \trna gene expression during mouse development}

Unlike protein-coding genes, \trna genes are not directly implicated in changes
of cellular function. We therefore did not expect many changes of the levels of
\trna gene expression over the course of development. We do in fact observe that
many \trna gene expression levels remain stable (\cref{fig:trna-counts}).
Nevertheless, we also observe that around \num{50} per cent of all \trna genes
\emph{are} differentially expressed. \Cref{fig:liver-trna-expression-change}
shows a genomic location containing \trna genes which display these different
dynamics.

\textfigtwo{trna-counts}
    {Overview over \trna gene expression change.}
    {Bar plots show different types of \trna gene expression dynamics: \trna
    genes without change in their expression levels, \trna genes with changes to
    their expression levels, which are nevertheless expressed in all stages of
    development across both tissues; and \trna genes which are only expressed in
    a subset of all conditions.}
    {liver-trna-expression-change}
    {Example of dynamically changing \trna genes.}
    {Genomic region showing different types of \trna gene expression behaviour;
    the label colours on the x axis corresponds to the colours in
    \cref{fig:trna-counts}}

Surprisingly, the \trna gene expression differences follow similar patterns to
those observed in \mrna genes when plotting the first two principal components
of their \pca (\cref{fig:trna-pca}). The observed patterns are incompatible with
mere \emph{random} expression changes. Something must account for these
concerted changes in \trna gene expression.

\textfig{trna-pca}{body}{0.5\textwidth}
    {\pca of \trna gene expression per developmental stage.}
    {Rotations \num{1} and \num{2} of the correlation matrix of
    \trna gene expression in each developmental stage. The percentage on the
    axes shows the amount of variance explained by each rotation.}

In the case of the protein-coding genes, we can explain the nonrandom changes in
gene expression by known gene regulatory mechanisms, which control the
transcriptome of each cell and developmental stage. The fact that \trna gene
expression changes across development exhibit the same patterns as \mrna gene
expression changes suggests that \trna gene expression is subject to similar
regulatory constraints. We therefore attempted to explain \emph{why} \trna gene
expression requires changing in a regulated manner, and \emph{how} this \trna
gene regulation is carried out by the cell.

Our first suspicion was that changes in \mrna gene expression might lead to
changes in codon demand, since different protein-coding genes are made up from
different codons. The change in codon demand in turn could lead to a change in
anticodon supply in the form of differential \trna gene expression. This would
meet the need for efficient translation: mismatching codon and anticodon pools
would either lead to a wasteful over-production of \trna[s] of a given
anticodon, or to a bottleneck in such an anticodon supply, causing efficiency
loss in translation. Both scenarios present a non-optimal scenario for the
fitness of the cell and should reasonably be selected against.

We therefore went on to quantify the codon pool corresponding to each given
transcriptome\footnote{I shall refrain from using the term “codome” to refer to
the -ome of codons}, as well as the pool of available \trna genes, grouped by
their anticodon isoacceptor identity.

\section{Every mouse \mrna transcriptome encodes the same distribution of
triplet codons and amino acids}

The codon pool of a given \mrna transcript is given by the destribution of
triplet codons in its sequence. There are 64 possible triplet codons, of which
61 encode 20 different amino acids, and three encode the stop codon, marking the
end of translation.\footnote{Our analysis ignores selenocysteine, which is a
\num{21}st possible amino acid, and which can be encoded by one of the stop
codons under very rare circumstances.} Using this information and the transcript
abundance, quantified by our \rnaseq data, we calculated the abundance of each
triplet codon as well as each amino acid in the transcriptome of each
developmental stage. We then compared these across stages to find out how they
varied.

We find that on the transcriptome level, codon abundance is highly stable across
development in both tissues (Spearman’s \(\rho > 0.97\)) ---
\cref{fig:codon-anticodon-abundance} top left shows this by way of example using
the codons for arginine in the different stages in liver.

Given this stability, we wondered how much variation should be expected, by
simulating random transcriptomes, and computing their codon usage.
\Cref{fig:codon-anticodon-abundance} top middle and right shows that even for
simulated transcriptomes the codon usage remains unchanged. In fact, both
observed and simulated transcriptomes seem to simply reflect the codon abundance
found in the coding part of the genome, regardless of the sometimes strong
variations in gene expression between different transcriptomes.

\textfig{codon-anticodon-abundance}{spill}{0.8\textwidth}
    {Codon and anticodon abundance across stages of development.}
    {The figure consists of three panels (right, middle, left) with three
    subfigures (top, centre, bottom) each. The left panel shows observed data
    for each of the developmental stages in liver (brain data is comparable).
    The middle and right panels show simulated data from randomised
    transcriptomes. The middle panel used only expressed genes of each
    respective stage in the simulated data, whereas the right panel uses all
    genes, also unexpressed ones). The top figures of each panel show relative
    \mrna transcript triplet codon usage, using the representative example of
    arginine. The centre figures show the relative \trna anticodon abundance of
    the arginine isotype family. The bottom right figure shows the linear
    regression of relative codon usage against relative anticodon abundance in
    liver E15.5, along with its Spearman rank correlation. Triplet codons
    without directly corresponding anticodon (grey dots) were ignored in the
    calculation. The bottom middle and right figure shows the Spearman rank
    correlation coefficient of each stage’s relative codon and anticodon
    abundance (diamond), and the range of correlation coefficients for the
    simulated codon and anticodon pools (box plots) for each stage.}

\section{Stable isoacceptor anticodon abundance through development indicates
tight regulation of tRNA gene expression}

Next, we looked at the abundance of the matching \trna isoacceptors by summing
the expression of all \trna genes belonging to the same isoacceptor family.
\Cref{fig:codon-anticodon-abundance} centre left shows the relative anticodon
isoacceptor abundance of the arginine isotype family. Again we find that the
abundance stays stable across development in both tissues (Spearman’s \(\rho >
0.96\)).

In the same way as for the \mrna transcriptome, we then simulated random \trna
transcriptomes and calculated the relative abundance of all isoacceptor
families. Unlike for the observed \trna data, we find that simulated \trna
transcriptomes create highly variable pools of anticodon isoacceptors
(\cref{fig:codon-anticodon-abundance} centre middle and right). The variability
observed here is explained by the fact that there are only \num{433} \trna genes
in \mmu (of which only \num{311} were expressed in our samples), compared to the
approximately \num{20000} protein-coding genes, which leads to a bigger
influence of random sampling. In contrast to \mrna gene expression, the stable
anticodon isoacceptor abundance distribution we observe across development is
therefore not compatible with random variation in the \trna gene expression:
instead, it demonstrates the necessity of a mechanism actively stabilising \trna
gene expression variation at the anticodon isoacceptor level.

\section{\mrna triplet codon usage is highly correlated with \trna anticodon
isoacceptor abundance during development}

Codons and anticodon-carrying \trna[s] form the biochemical interface between
the genetic code and the amino acid sequence of proteins during \mrna
translation. We investigated this correspondence between \mrna-driven codon
demand and \trna anticodon supply by looking at the correlation of codons’ usage
in an \mrna transcriptome and its matching \trna anticodon isoacceptor
abundances in matching stages of development
(\cref{fig:codon-anticodon-abundance} bottom left). Across both tissues and all
stages of development, we find that \mrna codon demand and \trna anticodon
isoacceptor abundance are highly correlated (\(0.64 < \rho \leq 0.76\)
Spearman’s rank correlation, all \(p < 0.001\); the calculation excluded
unmatched \mrna codons due to wobble base pairing\footnote{This method was used
mainly for of simplicity; \cref{sec:trna-analysis} contains a discussion of more
accurate measures of the match between codon demand and \trna anticodon supply;
however, while this changes the numbers we find, these changes don’t affect the
conclusion.}).\todo{Mention correlation in pools of highly/lowly expressed \mrna
genes?}

We can compare these correlations between \mrna codon demand and \trna anticodon
supply with the correlations we find between our simulated \mrna and \trna
transcriptomes. In fact, correlating all \num{100} randomly simulated \trna
transcriptomes per tissue with the simulated \mrna transcriptomes yields a
distribution of significantly lower rank correlation coefficients
(\cref{fig:codon-anticodon-abundance} bottom middle and right).

This result provides further evidence that random variation of \trna gene
expression cannot account for the observed patterns of expression, and that
\trna gene expression must be actively regulated to stabilise the steady
abundance of the of \trna anticodon isoacceptors, matching the triplet codon
demand of the \mrna transcriptome.

\section{\trna anticodon isoacceptor families are transcriptionally compensated
across development}

Having established that \trna gene expression varies in a controlled fashion, we
were interested in finding out more about the mechanism driving this variation.
From what we know about the regulation of protein-coding genes, it seemed likely
that local genomic features around each \trna gene would be implicated in its
transcriptional regulation.

Previously published results indicate that there is no clear relationship
between sequence variation of the internal promoters of \trna gene and their
expression levels \citep{Oler:2010,Canella:2012}. We therefore focussed on the
sequence upstream of the \tss of \trna genes to search for \emph{cis}-regulatory
regions.

\textfloat{compensation}{spill}{%
    \centering
    \begin{minipage}{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{correlation-cag}
        \subcaption{Isoacceptor CAG \trna gene expression levels}
    \end{minipage}
    \hspace{0.05\textwidth}%
    \begin{minipage}{0.4\textwidth}
        \centering
        \includegraphics[width=\textwidth]{compensation-cag}
        \subcaption{Density curve of isoacceptor CAG \trna gene expression
        correlations}
    \end{minipage}
    \par
    \begin{minipage}{0.45\textwidth}
        \centering
        \includegraphics[width=\textwidth]{correlation-gcc}
        \subcaption{Isoacceptor GCC \trna gene expression levels}
    \end{minipage}
    \hspace{0.05\textwidth}%
    \begin{minipage}{0.4\textwidth}
        \centering
        \includegraphics[width=\textwidth]{compensation-gcc}
        \subcaption{Density curve of isoacceptor GCC \trna gene expression
        correlations}
    \end{minipage}}
    {\trna gene expression is compensated at the anticodon isoacceptor level
    during mouse development.}
    {…}
