\chapter{Analysis challenges concerning tRNA expression data}

We quantified \trna gene expression via \pol3 \chipseq. The reason for using
this, on the first glance indirect, measure is caused by the fact that \trna
genes are unfortunately not identifiable by their sequence alone: performing a
multiple sequence alignment of \trna genes in \mmu reveals that several \trna
genes share the exact same sequence.

\textfigure{trna-alignment}
    {{\footnotesize\input{data/mm10-trna-cove-asn.tex}}}
    {Alignment of Asn \trna genes.}
    {Parts of a multiple sequence alignment of \trna genes in \mmu generated
    with COVE\@. Shown are the \trna genes coding for Asn.}

In order to identify individual \trna genes and quantify their expression, we
therefore cannot resort to conventional \rnaseq: the \rna reads covering only
the transcribed gene region are indistinguishable.

With the \pol3 \chipseq data, we do not have this problem: reads cover both the
transcribed sequence and the flanking regions of each gene
(\cref{fig:trna-pol3-binding-profile}).

\textfig{trna-pol3-binding-profile}{\trna \pol3 \chip binding profile.}
    {The shaded, bell-shaped area shows an idealised binding profile of \chipseq
    data spanning the \trna gene, as well as its flanking regions upstream
    and downstream of the gene body. This overlap plays a role in identifying
    the individual gene.}

When mapping the reads, we therefore do not discard all non-uniquely mapping
reads as is otherwise customary.\footnote{We still discard reads which are
likely \pcr duplicates, i.e.\ map to many locations; we more or less arbitrarily
used the threshold of \num{50} non-unique mapping locations} We thus end up with
reads which have not been assigned to a given \trna gene
(\cref{fig:trna-pol3-map-ambiguous-reads-1}). In order to assign these reads to
\trna genes, we \emph{reallocated} reads after mapping, using the number of
uniquely mapping reads in \trna genes’ flanking regions to determine the most
likely origin (\cref{fig:trna-pol3-map-ambiguous-reads-2}).

\textfig{trna-pol3-map-ambiguous-reads-1}{Mapping ambiguous \chip reads.}
    {\chip reads originating from \trna genes can often not be mapped
    unambiguously to any given \trna. Instead, information form the gene’s
    flanking regions is used to determine the more likely provenance.}

\textfig{trna-pol3-map-ambiguous-reads-2}{Mapping ambiguous \chip reads.}
    {\chip reads originating from \trna genes can often not be mapped
    unambiguously to any given \trna. Instead, information form the gene’s
    flanking regions is used to determine the more likely provenance.}

\section{… all the rest}

\section{An overview over failed analysis approaches}
